File: projeto/UI/pinguino/src/app/models/endereco.ts
import { Municipio } from "./municipio";

export class Endereco {

    constructor() { 
        this.id = 0;
        this.logradouro = '';
        this.ativo = true;
        this.municipio = new Municipio();
        this.complemento = '';
        this.cep = '';
        this.numero = '';
    }

    id: number;
    logradouro: string;    
    numero: string;
    complemento: string;
    cep: string;
    municipio: Municipio;
    ativo: boolean;
}

File: projeto/API/PinguinoApp/PinguinoApp.API/Models/Endereco.cs
namespace PinguinoApp.API.Models
{
    public class Endereco
    {
        public int Id { get; set; }

        public string Logradouro { get; set; }

        public string Numero { get; set; }

        public string Complemento { get; set; }

        public Municipio Municipio { get; set; }

        public string Cep { get; set; }

        public bool Ativo { get; set; }
    }
}
